commet/lib/client/matrix/matrix_room.dart

```diff
@@ -124,7 +127,8 @@ class MatrixRoom extends Room {
   String get identifier => _matrixRoom.id;
 
   @override
-  Timeline? get timeline => _timeline;
+  Timeline? get timeline =>
+      _timeline ?? (_timeline = MatrixTimeline(client, this, matrixRoom));
```

Comment:
Probably should make this not just instantiate, and move to a different function. Otherwise we cant ever check if timeline exists without instantiating it